geo/src/main/java/org/odk/collect/geo/GeoPointActivity.java

```diff
@@ -94,15 +88,6 @@ protected void onCreate(Bundle savedInstanceState) {
             numberOfAvailableSatellites = savedInstanceState.getInt(NUMBER_OF_AVAILABLE_SATELLITES);
         }
 
-        Intent intent = getIntent();
-
-        targetAccuracy = DEFAULT_LOCATION_ACCURACY;
```

Comment:
This use of the default was redundant, as we pass the default if no accuracy has been specified through the extra. It felt like the accuracy default was  a `collect_app` concept rather than a `geo` concept, so now the Activity just defaults to the max `Double` value meaning there is effectively no threshold.